src/utils/url.ts

```diff
@@ -11,15 +11,21 @@ export const venueInsideUrl = (venueId: string) => {
   return `/in/${venueId}`;
 };
 
-export const adminNGVenueUrl = (venueId?: string) =>
-  generatePath("/admin-ng/venue/:venueId?", {
-    venueId: venueId,
-  });
-
-export const adminNGSettigsUrl = (venueId?: string) =>
-  generatePath("/admin-ng/advanced-settings/:venueId?", {
-    venueId: venueId,
-  });
+const createAdminUrlHelperFor = (segment: string) => (
+  venueId?: string,
+  selectedTab?: string
+) =>
+  segment
+    ? generatePath(`/admin-ng/:segment?/:venueId?/:selectedTab?`, {
+        segment: segment,
+        venueId: venueId,
+        selectedTab: selectedTab,
+      })
+    : generatePath("/admin-ng");
+
+export const adminNGRootUrl = createAdminUrlHelperFor("");
```

Comment:
The parameter can be optional and there would be no need for this empty string.

Reply:
Could, just looked a bit like:

`createAdminUrlHelperFor()` - create admin URL helper for ...
`createAdminUrlHelperFor("")`  - create admin URL helper for empty path

🤷 either way is fine by me, so unless specifically asking for the change, I'll leave it as it is

Reply:
Replaced with a nullary function for brevity and type signature @ 2c33b6b8de126d7a4550fdd2ca5b20e1ad2b5168